ui/src/backoffice/components/Author/PrivateNotes.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ContentBox from '../../../common/components/ContentBox';
+
+type PrivateNotesProps = {
+  privateNotes: { get: (arg0: string) => {} | null | undefined }[]; // TODO: define proper type for privateNotes
+};
+
+const PrivateNotes = ({ privateNotes }: PrivateNotesProps) => (
+  <ContentBox className="mb3" fullHeight={false} subTitle="Notes">
+    <i>
+      {privateNotes.map((note) => {
+        const value = note?.get('value');
+        return (
+          <p className="mb0" key={value as string}>
+            &quot;{value}&quot;
```

Comment:
since we are refactoring, why not `"`?

Reply:
Seems like yarn build fails in unescaped characters